code/clue.js

```diff
@@ -1,64 +1,210 @@
-// CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
-const mrGreen = {
-  firstName: "Jacob",
-  lastName: "Green",
-  color: "green",
-  description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
-  occupation: "Entrepreneur"
-}
-
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
+// OBJECTS FOR ALL THE SUSPECTS:
 
-const rope = {
-  name: "rope",
-  weight: 10
-}
+ const mrGreen = {
+    firstName: "Jacob",
+    lastName: "Green",
+    color: "green",
+    description: "He has a lot of connections",
+    age: 45,
+    image: "assets/green.png",
+    occupation: "Entrepreneur"
+  }
+ const professorPlum = {
+    firstName: "Victor",
+    lastName: "Plum",
+    color: "purple",
+    description: "Master detective",
+    age:"38",
+    image: "assets/plum.png",
+    occupation: "professor"
+  }
+ const missScarlet = {
+    firstName: "Cassandra",
+    lastName: "Scarlet",
+    color: " crimson",
+    description: "will do anything to stay in the sportight",
+    age:"25",
```

Comment:
So an age is a number, which means it is an integer in code. Now you have written it as a string (when you have `"` around) which you do if you are typing out letters in some way. So in this case you can re write this as `age: 25,` (and all of the others integers)